Cargo.toml

```diff
@@ -22,6 +22,7 @@ base64 = "0.13.0"
 rand = "0.8.2"
 serde = { version = "1.0.119", features = ["derive"] }
 serde_json = "1.0.61"
+serde-aux = "*"
```

Comment:
This needs to be a versioned dependency, so that it at least follows semver to avoid breakages. Just set it to the latest version of serde-aux `2.1.1` assuming it work.

Reply:
Done